### PR TITLE
Card

### PR DESCRIPTION
**Pull request by JatuFaique**
Please review the card component

### CODE REVIEW

File: components/Card.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  --black-chocolate: hsla(57, 100%, 4%, 1);
+}
+
+.disp-flex-col {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 8px;
```
Comment (NeerajVodala) [inline components/Card.css:28]: Try to be consistent with units (using rem would be recommended)

File: components/Card.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  column-gap: 2rem;
+  row-gap: 2rem;
+}
+
+.card {
+  overflow: hidden;
+  background-color: rgb(224, 224, 224);
```
Comment (NeerajVodala) [inline components/Card.css:42]: using variable would be recommended (so that you can reuse it)

**Review by NeerajVodala — COMMENTED** [review]
I see a lot of empty spaces in HTML file(can be refactored)
Inconsistent use of units(either use rems or px)
Rest everything looks good to me.
All the best.

**Comment by NeerajVodala** [comment]
> I see a lot of empty spaces in HTML file(can be refactored) Inconsistent use of units(either use rems or px) Rest everything looks good to me. All the best.

Also use variables for colors (so that you can reuse them easily)
Try to make separate file for variables

File: components/Card.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+                    </div>
+                </div>
+                
+               
+            </div>
+            <div class='btn-list'>
+                
+                <h3>Go Back</h3>
+            </div>
+
```
Comment (chanchal16) [inline components/Card.html:143]: remove these unwanted blank spaces. The rest looks good.

File: components/Card.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+* {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  --fulvous: rgb(217, 126, 28);
```
Comment (chanchal16) [inline components/Card.css:12]: You can store these variables in a separate file.

**Pull request merged**